config/routes.rb

```diff
@@ -1,5 +1,12 @@
 Rails.application.routes.draw do
+  require 'sidekiq/web'
```

Comment:
I know that i should be removing those on production environment but wanted to make my reviewers life a lil bit easier to navigate to sidekiq queue if needed 👍